src/hooks/auth0-provider.tsx

```diff
@@ -16,10 +25,10 @@ const initialState = {
 /**
  * @ignore
  */
-const getIdTokenProfileClaims = idToken => {
-  const payload = jwt_decode(idToken);
+const getIdTokenProfileClaims = (idToken: string): User => {
+  const payload: {[key: string]: any} = jwtDecode<JwtPayload>(idToken);
 
-  const profileClaims = Object.keys(payload).reduce((profile, claim) => {
+  const profileClaims = Object.keys(payload).reduce((profile: any, claim) => {
```

Comment:
I don't think `profile` needs to be `any` here? Don't we want `Partial<User>` ?

Reply:
This is a nifty feature I didn't about in TS, nice +1